backend/src/main/java/com/staccato/member/domain/Nickname.java

```diff
@@ -0,0 +1,46 @@
+package com.staccato.member.domain;
+
+import java.util.regex.Pattern;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embeddable;
+
+import com.staccato.exception.StaccatoException;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+@Embeddable
+@EqualsAndHashCode
+public class Nickname {
+    public static final Pattern NICKNAME_REGEX = Pattern.compile("^[ㄱ-ㅎㅏ-ㅣ가-힣a-zA-Z._]+$");
```

Comment:
```suggestion
    private static final Pattern NICKNAME_REGEX = Pattern.compile("^[ㄱ-ㅎㅏ-ㅣ가-힣a-zA-Z._]+$");
```

혹시 다른 클래스에도 해당 정규식 패턴을 사용하는 코드가 있을까요? 없다면 `private`이 좋을 것 같습니다.

Reply:
놓쳤네요! 짚어주셔서 고맙습니다👍